### PR TITLE
Added tutorial on Supervised Word Mover Distance

### PR DESCRIPTION
**Pull request by gasbas**
Project for the OT course ENSAE 2022. 
Authors: COURRIER Violaine, MICHEL Gaspard

### CODE REVIEW

File: docs/notebooks/Supervised_WMD.ipynb [hunk 1]
```diff
@@ -0,0 +1,1441 @@
+{
```
Comment (marcocuturi) [inline docs/notebooks/Supervised_WMD.ipynb:1]: <p>This should be shorter, and you can remove the authors' name and year (since the the goal is to incorporate in a documentation)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (marcocuturi) [inline docs/notebooks/Supervised_WMD.ipynb:1]: <p>since this is copypasted from (https://ott-jax.readthedocs.io/en/latest/notebooks/One_Sinkhorn.html) you can remove this entire section and refer to the colab in the doc, and start from your contribution</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (marcocuturi) [inline docs/notebooks/Supervised_WMD.ipynb:1]: <p>if this is not conclusive it's going to be difficult to add it as a tutorial to OTT :) (of course you can be graded) do you have something more conclusive?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Comment (gasbas) [reply docs/notebooks/Supervised_WMD.ipynb:1]: <p>Thanks for the comments ! Unfortunately, we do not have more conclusive results for the moment. As said, we believe that increasing the batch size and the number of neighborhoods would lead to better results. However, increasing those two parameters leads to a Jax OOM error. Snce we discovered Jax library through the project, we do not have enough knowledge to understand why for the moment. We believe it is due to taking the gradient (through <code>jax.value_and_grad</code> function)  of a jitted function. A better replacement might be to jit the value and grad function directly, but doing so leads to a ConcretizationTypeError that we do not understand how to solve. </p><p> </p><p>We still have some projects due for this academic year, but maybe starting from mid May I could take a closer look and try to overcome this ! </p>

**Comment by marcocuturi** [comment]
LMK if you would like to reopen this.
